Content.Server/Geras/GerasSystem.cs

```diff
@@ -29,13 +30,22 @@ private void OnMapInit(EntityUid uid, GerasComponent component, MapInitEvent arg
 
     private void OnMorphIntoGeras(EntityUid uid, GerasComponent component, MorphIntoGeras args)
     {
+        if (HasComp<ZombieComponent>(uid))
+            return; // i hate zomber.
+
```

Comment:
Can't you just sub to zombie and remove the action instead of leading a dummy action on them indefinitely.